spec/features/users/user_updates_profile_spec.rb

```diff
@@ -7,11 +7,11 @@
 
     visit edit_users_path
 
-    fill_in "user_email", with: "JT@example.com"
-    fill_in "user_name", with: user.name
+    fill_in "user_name", with: "Fred"
+    fill_in "user_password", with: user.password
```

Comment:
I know I mentioned this before. Adding this field here, means a user cannot update their profile without inputing in their password. Side note: suggesting to fix in the future.

Reply:
Sorry, slow on the uptake, will fix in another PR!